config.toml.example

```diff
@@ -29,11 +30,11 @@ theme = "indigo"
 
   # These are parameters used for indieweb identity. You should set these along
   # with the above email/social network parameters.
-  [params.indieWeb]  
+  [params.indieWeb]
```

Comment:
@AngeloStavrow in this commit there are some changes which have nothing to do with the PR. Are you okay by merging them? I noticed some trailing whitspaces in some files, but had not the time to fix this. Maybe a new issue is a good idea for this?

Reply:
Opening a new issue is a good idea. I haven't really done a thorough audit of whitespace issues.

In the meanwhile I think it's fine to include these changes with this PR as it's really just a bit of cleanup, and not a change that affects functionality in some unrelated way.